cards/eyeddx

```diff
@@ -0,0 +1,31 @@
+title: See the Unseen: Use the Rule of 2s to Never Miss Another Ocular Emergency
+author:
+    - Puja Gopal, MD
+created: 2017/04/14
+updates: null
+categories:
+    - Ophthalmology
+ 
+---
+# Background
+
+Eye complaints are a common chief complaint in the emergency department (ED). Approximately 3.3% of patient visits to the ED were related to an eye or ear complaint in 2005 1. Having a systematic approach to ocular complaints is especially important as each structure of the eye holds within it different pathology 2, 3. It is often difficult to remember individual diagnoses for various complaints such as red eye, painful eye, or vision loss, especially during busy shifts. The rule of 2s, as illustrated below, is a simple method with an anatomical approach to major ocular diagnoses encountered in the ED. It ensures one doesn't miss clinically significant diagnoses and requires only a basic understanding of the anatomy of the eye. The rule of 2s is easy to remember as it contains the number 2 in the context of eyes and proceeds from layer to layer of the eye in an anterior to posterior direction highlighting the 2 major associated diagnoses of that layer.  
+This approach is not meant to be all-inclusive of ocular diagnoses but rather is meant to provide a framework one can use to guide the physical exam, such as while using the slit lamp, and to systematically include or eliminate certain diagnoses.
+
+## ANTERIOR EYE – utilize slit lamp exam
+![](image-1.png)
+
+## POSTERIOR EYE – use fundoscopy and ultrasound
+![](image-2.png)
+
+## OTHER EYE DIAGNOSES
+![](image-3.png)
+
+# REFERENCES
```

Comment:
Should be H2 
```
## References
```
See the style guide: https://github.com/aliemteam/aliemcards/wiki/Markdown-Style-Guide